src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -83,6 +116,33 @@ public void setNumberOfChannels(String mediaInfoValue) {
 	}
 
 	/**
+	 * Get bits per sample for this audio track.
+	 * @return bits per sample (default is 16)
+	 */
+	public int getBitsperSample() {
+		return bitsperSample;
+	}
+
+	/**
+	 * Set bits per sample for this audio track.
+	 * @param bitsperSample bits per sample to set
+	 */
+	public void setBitsperSample(int bitsperSample) {
+		if (bitsperSample < 1) {
+			throw new IllegalArgumentException("Bits per sample can't be less than 1.");
+		}
```

Comment:
@Sami32 I've pushed a commit where I've shown what I mean for ```bitRate```. I've also simplified the regex matching, now it will use the leftmost number value in the string.